Cpp/odin-views/app_secure.cpp

```diff
@@ -13,12 +13,23 @@
 
 #include <fost/crypto>
 #include <fost/insert>
+#include <fost/json>
 #include <fost/log>
 #include <fostgres/sql.hpp>
 
 
 namespace {
 
+    inline std::pair<boost::shared_ptr<fostlib::mime>, int>
+            respond(fostlib::string message, int code = 403) {
+        fostlib::json ret;
+        if (not message.empty())
+            fostlib::insert(ret, "message", std::move(message));
+        fostlib::mime::mime_headers headers;
+        boost::shared_ptr<fostlib::mime> response(new fostlib::text_body(
+                fostlib::json::unparse(ret, true), headers, "application/json"));
+        return std::make_pair(response, code);
+    }
```

Comment:
On Line#43: `jwt_iss.find(odin::c_app_namespace.value())` -> Should use `starts_with` instead of `find`